### PR TITLE
Test Travis CI with Trusty image.

### PR DESCRIPTION
**Pull request by jmitchell**
See https://blog.travis-ci.com/2017-04-17-precise-EOL for motivation.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+dist: trusty
```
Comment (deepfire) [inline .travis.yml]: Is this really what we want?  It's currently `precise`, AIUI.

After reading the blog completely, I'm confused:
- per Travis job warning we have: `This job ran on our Precise, sudo: required environment which will be updated on May 3rd. You can check more details about this in our blog.` -- that's what sent me into the panic mode
- per the blog post we have: `We’re focusing on making the switch to our default stable image from Ubuntu Precise 12.04 to Ubuntu Trusty 14.04 in early Q3, sometime in July.`

**Comment by deepfire** [comment]
After reading the blog post completely, I'm confused as to how much time do we have:
- per Travis job warning we have: `This job ran on our Precise, sudo: required environment which will be updated on May 3rd. You can check more details about this in our blog.` -- that's what sent me into the panic mode
- per the blog post we have: `We’re focusing on making the switch to our default stable image from Ubuntu Precise 12.04 to Ubuntu Trusty 14.04 in early Q3, sometime in July.`

**Comment by jmitchell** [comment]
Travis' timeline is confusing I agree. Hopefully this will just work.

Regarding testing in a separate branch, this PR corresponds to a new branch off of `master`. We shouldn't merge it until CI is passing and the output is verified.

**Comment by domenkozar** [comment]
I pushed another test out of current master, due to https://blog.travis-ci.com/2017-07-11-trusty-as-default-linux-is-coming

**Comment by domenkozar** [comment]
We have our jobs now on trusty (as it says in yellow on build page) so it works.

**Pull request closed without merging**